packages/@ember/-internals/glimmer/lib/components/input.ts

```diff
@@ -38,7 +38,13 @@ import InternalComponent from './internal';
 
   ### Actions
 
-  The `Input` component takes a number of arguments with callbacks that are invoked in response to
+  There are multiple ways to pass actions to the input component. Starting with Ember Octane, we recommend using the `{{on}}` modifier to call an action on specific events such as the input event.
+
+  ```handlebars
+  <Input @value={{this.name}} {{on "input" this.validateName}} />
+  ```
+
+  The `Input` component also takes a number of arguments with callbacks that are invoked in response to
   user events.
 
   * `enter`
```

Comment:
```suggestion
  * `enter` (synthetic event)
```

IMO we should do this for this for all the synthetic events and at the end of the list mention what they are (something along the lines of "Some of these do not correspond to a DOM event. Instead, they are synthetic/logical events that are fired by the `Input` component when a high-level user action is take (such as pressing the Enter key).")